env.default

```diff
@@ -8,6 +8,9 @@ TARGET_DOMAINS=foundation.mozilla.org
 
 # network-api environment:
 ALLOWED_HOSTS=*
+# APP_ENVIRONMENT is used to prepend [S] or [RA] to page tab title & change favicon color,
+# choices are "Review" and "Staging" and are set in Heroku.
+APP_ENVIRONMENT=
```

Comment:
If these are auto-set by heroku, we don't need to add them to the default env, but we do want to mention them in our docs that talk about the various environment variables.

Reply:
Hi Pomax! These are not auto-generated by heroku, we had to manually set them in both the review app settings and the staging settings